modules/ddl-and-loading/pages/defining-a-graph-schema.adoc

```diff
@@ -229,23 +230,27 @@ image::comms-graph-outdegree.png["Diagram of a graph with five person vertices.
 `CREATE EDGE` defines a new global edge type. Data loaded to a global edge type in one graph will be shared across all graphs that use that edge type.
 See xref:modifying-a-graph-schema.adoc#_global_vs_local_schema_changes[Global vs. local schema changes] for instructions on creating local edges.
 
-There are two forms of the `CREATE EDGE` statement, one for directed edges and one for undirected edges.
-Each edge type must specify that it connects FROM one vertex type TO another vertex type.
-Additional pairs of `FROM` and `TO` vertex types may be added.
-Then additional attributes may be added.
-Each attribute follows the same requirements as described in the Attribute List subsection for the `CREATE VERTEX` section.
+A `CREATE EDGE` statement must specify at least one pair of source and target vertex types, that is, the vertex types on both ends of the edge.
+In GSQL, they are often referred to as `FROM` vertex and `TO` vertex, respectively, to reflect the keywords used to define the edge type.
+
+By default, only one instance of an edge type is allowed between two vertices.
+You can add a discriminator to the edge type definition to allow for multiple instances of one edge type between two vertices.
+
 
 [tabs]
 ====
 CREATE UNDIRECTED EDGE::
 +
 --
-[source,ebnf]
+[source.wrap,ebnf]
 ----
 CREATE UNDIRECTED EDGE Edge_Type_Name "("
         FROM Vertex_Type_Name "," TO Vertex_Type_Name
    ["|" FROM Vertex_Type_Name, TO Vertex_Type_Name]*
-   ["," attribute_name type [DEFAULT default_value]]* ")"
+   [ ( attribute_name type DISCRIMINATOR ) |
```

Comment:
remove this line, we do not support DISCIMINATOR keywords after attribute_name type